cli/kv.go

```diff
@@ -286,6 +286,33 @@ func runReverseScan(cmd *cobra.Command, args []string) {
 	showResult(rows)
 }
 
+var rangeDelCmd = &cobra.Command{
+	Use:   "rangedel [options] <startKey> <endKey>",
```

Comment:
There's a TODO at the top of this file mentioning DeleteRange - could you remove that bit?